command-parser.js

```diff
@@ -113,6 +113,21 @@ class CommandContext {
 		this.inputUsername = '';
 	}
 
+	checkFormat(room, message) {
+		if (!room || !room.stretching && !room.capitals) return false;
```

Comment:
`a || b && c` is frowned upon in certain circles.

You might as well make this two lines.